apps/cms.c

```diff
@@ -869,16 +896,20 @@ int cms_main(int argc, char **argv)
             goto end;
     }
 
-    in = bio_open_default(infile, 'r', informat);
+    warn_binary(infile, flags);
```

Comment:
maybe this should be optional?

Reply:
I strongly recommend having this warning by default in case a user forgot to use `-binary`
because unsuspecting users would forget to activate that warning, too ;-)